components/service/glean/src/test/java/mozilla/components/service/glean/scheduler/MetricsPingSchedulerTest.kt

```diff
@@ -19,6 +19,7 @@ import java.util.concurrent.TimeUnit
 class MetricsPingSchedulerTest {
     // NOTE: Using a "real" ApplicationContext here so that it will have
     // a working SharedPreferences implementation
+    @Suppress("DEPRECATION")
     private val metricsPingScheduler = MetricsPingScheduler(RuntimeEnvironment.application.applicationContext)
```

Comment:
I think we can pass in `ApplicationProvider.getApplicationContext()`

Reply:
Indeed.  That works.